src/components/OrderPage/QuantityButtons/index.js

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react"
+import './QuantityButtons.scss'
+
+const QuantityButtons = ({ foodName, price, addItem, removeItem, orderObject }) => {
+    const [quantity, setQuantity] = useState(0)
+
+    function findQty() {
+        let itemInOrder = false
+        orderObject.forEach(orderItem => {
+            if (orderItem.foodName === foodName) {
+                itemInOrder = true
+                setQuantity(orderItem.qty)
+            }
+        })
+        if (!itemInOrder){
+            setQuantity(0)
+        }
+    }
+    useEffect(findQty, [orderObject])
+
+    return (
+        <div className="quantityButtons">
+            <button className="quantityButtons__btn" onClick={() => {
+                addItem(foodName, price)
+                findQty()
+            }}>+</button>
+            <div className="quantityButtons__qty">{quantity}</div>
+            <button className="quantityButtons__btn" onClick={() => {
+                removeItem(foodName, price)
+                findQty()
```

Comment:
No need to call this here - `removeItem()` updates the order state, which then causes the above use effect to run and update the qty